workers_launch_template_mixed.tf

```diff
@@ -316,6 +316,21 @@ resource "aws_launch_template" "workers_launch_template_mixed" {
     }
   }
 
+  tag_specifications {
+    resource_type = "volume"
+
+    tags = merge(
+      {
+        "Name" = "${aws_eks_cluster.this.name}-${lookup(
+          var.worker_groups_launch_template[count.index],
```

Comment:
There was a typo here I missed. Should be `var.worker_groups_launch_template_mixed[count.index],`. Fixed in latest release though 😃

https://github.com/terraform-aws-modules/terraform-aws-eks/pull/452